### ISSUE TITLE
Unselecting terrain surfaces makes last two surface types unselectable in land window

### ISSUE THREAD

**Issue by karst** [OpenRCT2/OpenRCT2#19293]
### Operating System

Windows 10, 64 Bit

### OpenRCT2 build

OpenRCT2, v0.4.3-216-gb1b056e (b1b056e on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2, RollerCoaster Tycoon Classic

### Describe the issue

![image](https://user-images.githubusercontent.com/1516893/215635330-a08c2437-8db7-4c48-a7e2-bff490851010.png)

When unselecting terrain surfaces, some terrain surfaces become unselectable. It seems to be the last two in the list.

### Steps to reproduce

1. Open a park
2. Go to object selector
3. Unselect some surfaces (for example the grids)
4. Try selecting the final two land types in the menu, it'll turn invisible and you'll get an error "Can't change land type..." when trying to paint it.

**Comment by tan-x-dx** [comment]
Duplicate of https://github.com/OpenRCT2/OpenRCT2/issues/18990

Shall I close mine?

**Comment by karst** [comment]
No, feel free to leave it open, good that you mention it. First one to report usually is left open and the duplicates are closed.
